sagemaker-debugger/xgboost_census_explanations/xgboost-census-debugger-rules.ipynb

```diff
@@ -7,7 +7,7 @@
    },
```

Comment:
<p>suggestions:</p><ul><li>modify the function <code>plot_feature_importance</code> so that in the legend of the plot should show "Age", "Workclass", "Race" (i.e the actual name of the features) instead of <code>feature_importance/cover/f0</code> etc.</li><li>the color of the curve for each feature should be distinct. If there are too many features on one plot, consider split into say 4 plots</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>fixed</p>